src/lib.rs

```diff
@@ -267,34 +270,16 @@ pub fn get_metadata(messages: &[Message]) -> CTRConfig {
     }
 }
 
-/// Builds the smdh using `smdhtool`.
-/// This will fail if `smdhtool` is not within the running directory or in a directory found in $PATH
-pub fn build_smdh(config: &CTRConfig, verbose: bool) {
-    let mut command = Command::new("smdhtool");
-    command
-        .arg("--create")
-        .arg(&config.name)
-        .arg(&config.description)
-        .arg(&config.author)
-        .arg(&config.icon)
-        .arg(config.path_smdh())
-        .stdin(Stdio::inherit())
-        .stdout(Stdio::inherit())
-        .stderr(Stdio::inherit());
-
-    if verbose {
-        print_command(&command);
-    }
+/// Builds the smdh using `cytryna` library.
+pub fn build_smdh(config: &CTRConfig) {
+    let smdh = Smdh::builder()
+        .with_short_desc(&config.name).unwrap()
```

Comment:
Is it possible the user could input something that would cause these unwraps to fail? Would the error be understandable enough to easily fix if so?

If not we could augment with an `expect` or `map_err` just to make it more clear, I guess.

Reply:
User could input a very long string for any of those which would error out and yeah `expect` for those is a good idea